map-storage/src-ui/App.svelte

```diff
@@ -80,15 +86,26 @@
             <p>Loading maps list...</p>
         {:then json}
             <h2>Maps list</h2>
-            <ul>
+            <table>
+                <tr>
+                    <th>
+                        WAM location
+                    </th>
+                    <th>
+                        map location
+                    </th>
+                </tr>
                 {#each Object.entries(json.maps) as [name, map]}
-                <li>
-                    <a href={mapStorageUrl + "/" + name} target="_blank">{name}</a>
-                    &rarr;
-                    <a href={map.mapUrl} target="_blank">{map.mapUrl}</a>
-                </li>
+                    <tr>
+                        <td>
+                            <a href={"../" + name} target="_blank">{name}</a>
```

Comment:
The "../" will only work in deployments where the map-storage is hosted on "/map-storage". In some deployments, it can be hosted on a different subdomain. Therefore, it would be better to rely on a custom environment variable pointing to the "PLAY_URL".

Reply:
Also: Why `PLAY_URL` and not `PUBLIC_MAP_STORAGE_URL`?